pro_tes/ga4gh/tes/task_runs.py

```diff
@@ -366,10 +368,17 @@ def cancel_task(self, id: str, **kwargs) -> dict:
                 f"{db_document.tes_endpoint.host.rstrip('/')}/"
                 f"{db_document.tes_endpoint.base_path.strip('/')}"
             )
-            if self.store_logs:
-                task_id = db_document.task.logs[0].metadata.forwarded_to.id
+
+            assert db_document.task.logs is not None
+            logs: list[TesTaskLog] = db_document.task.logs
+
+            if (self.store_logs and logs and logs[0].metadata and
+                    logs[0].metadata.forwarded_to):
```

Comment:
Ufff...... This is very hard to read/understand...

Reply:
lemme know if this fixes it :)